### DISCUSSION TITLE
blockscout is not able to read array returning functions.

### DISCUSSION THREAD

**Discussion by cryptonix21** [blockscout/blockscout#8872] [Q&A]
Hi @everyone

In the blockscout new UI return array values are showing undefined ? 

![array return](https://github.com/blockscout/blockscout/assets/59303056/a190c1de-d95c-4f0d-9665-e4a69761fb79)

Does anyone know how to resolve this issue and i'm using frontend v1.11.2.

**Comment by Mojmir-R** [comment]
v1.11.2 is 2 months old release, as a first thing I would advise you to try updating to the latest which is 1.17.1: https://github.com/blockscout/frontend/releases
